src/main/java/com/snoop/controllers/WorkspaceController.java

```diff
@@ -57,7 +57,7 @@ public String index(Map<String, Object> model) {
 
 		logger.debug("index() is executed!");
 
-		Workspace wkps = new Workspace(new Long(1));
+		Workspace wkps = new Workspace(Long.valueOf(1));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this "Long" constructor [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2129)